thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -1140,19 +1143,17 @@ public Trigger getTrigger(
         IoTRestRequest request = new IoTRestRequest(url, IoTRestRequest.Method.GET, headers);
         JSONObject responseBody = this.restClient.sendRequest(request);
 
-        //TODO: // FIXME: 2017/01/23
-        return null;
-//        Schema schema = null;
-//        JSONObject commandObject = responseBody.optJSONObject("command");
-//        if (commandObject != null) {
-//            String schemaName = commandObject.optString("schema", null);
-//            int schemaVersion = commandObject.optInt("schemaVersion");
-//            schema = this.getSchema(schemaName, schemaVersion);
-//            if (schema == null) {
-//                throw new UnsupportedSchemaException(schemaName, schemaVersion);
-//            }
-//        }
-//        return this.deserialize(schema, responseBody, this.target.getTypedID());
+        try {
+            Trigger trigger =  this.gson.fromJson(responseBody.toString(), Trigger.class);
+            TriggerUtils.setTargetID(trigger, this.target.getTypedID());
```

Comment:
@yongpingchen What do you think about adding target id to `responseBody`?

I think that using reflection is not good way in this situation.